lib/detect-testing-library-utils.ts

```diff
@@ -149,6 +154,10 @@ export function detectTestingLibraryUtils<
         originalNodeName?: string
       ) => boolean
     ): boolean {
+      if (!node) {
```

Comment:
Safety check. Apparently there was some place calling this method with `null` node. I need to investigate this properly, but I did this workaround meanwhile.